### ISSUE TITLE
[Feature Request] Add "verbose" option to polymer-build

### ISSUE THREAD

**Issue by FredKSchott** [Polymer/polymer-build#37]
We've always relied on this output from the CLI, lets support it in the build library as well

**Comment by FredKSchott** [comment]
Blocked by https://github.com/Polymer/polymer-build/pull/43

**Comment by web-padawan** [comment]
Any updates?

**Comment by FredKSchott** [comment]
nope, haven't had the time to prioritize this. The work itself is actually pretty straightforward, if anyone wants to take a stab at it let me know and I can give some additional context here.

**Comment by jared-unify** [comment]
Would appreciate more output on build. Far too often my project will work with serve but won't build properly and it doesn't say it hasn't built properly. Debugging is really hard.
